README.md

```diff
@@ -0,0 +1,22 @@
+
+[![PkgGoDev](https://pkg.go.dev/badge/github.com/vimeo/go-clocks)](https://pkg.go.dev/github.com/vimeo/go-clocks)
+
+## go-clocks provides a modern context-aware clock interface
+
+When writing code that involves time, it is not uncommon to want control over
+the flow of that resource. Many applications end up storing a
+field called `now` and of type `func() time.Time`, to mock out time, but that
+only covers a fraction of the `time` package's interface, and only covers the
+most trivial cases.
+
+The `Clock` interface in `go-clocks` covers both querying and waiting for time.
+
+### `FakeClock` and tests
+
+While `Clock` is a nice interface with both context-aware `SleepFor` and
+`SleepUntil`, it really shines when combined with the `FakeClock` in this
+repository. `FakeClock`
```

Comment:
We can remove `FakeClock` at the end of this paragraph.

Reply:
Removed. And Updated all references to `FakeClock` to reference `fake.Clock`